src/shared/parameter/config_definitions/tbots_gtest_main_command_line_args.yaml

```diff
@@ -16,3 +16,15 @@
     description: >-
         The directory to output logs to. Absolute paths are recommended as the working directory
         is inside the bazel-out directory.
+
+- double:
+    name: test_speed
+    value: 1.0
+    min: 0.1
+    max: 10.0
+    description: >-
+        The speed slow down factor.
```

Comment:
nit: would be good to expand on the description a bit here by saying 0.1 would be 10x slower than realtime and 10 would be 10x faster than realtime.

edit: ah the comment is below, but you should also add it here

because when someone runs the test with --help, this description will print on screen

Reply:
Added